<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLbsTrack</class>
 <widget class="QFrame" name="CLbsTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track</string>
  </property>
  <widget class="QTextBrowser" name="txtLogger">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>288</y>
     <width>582</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QQuickWidget" name="quickWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>582</width>
     <height>273</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>4</verstretch>
    </sizepolicy>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
